packages/core/src/node/backend-application.ts

```diff
@@ -137,6 +138,7 @@ export class BackendApplication {
 
     use(...handlers: express.Handler[]): void {
         this.app.use(...handlers);
+        this.app.use(serveGzip);
```

Comment:
I don't think that `BackendApplication.use` is a good method to install `serveGzip` handler, since it can be called multiple times and each time `serveGzip` will be installed.

Could you move installation `serveGzip` handler in the constructor?

Reply:
oh I removed that on my tree.